src/creator.rs

```diff
@@ -27,7 +27,7 @@ pub(crate) struct Creator<H: Hasher> {
     parents_rx: Receiver<Unit<H>>,
     new_units_tx: Sender<NotificationOut<H>>,
     n_members: NodeCount,
-    current_round: Round, // current_round is the round number of our next unit
+    current_round: Round, // current_round is the highest round number of all known units
     candidates_by_round: Vec<NodeMap<Option<H::Hash>>>,
     n_candidates_by_round: Vec<NodeCount>,
     create_lag: DelaySchedule,
```

Comment:
There is a lie in the documenting comment, that was a lie even before these changes but it's a good moment to fix it – we don't create units as soon as we can, but according to a delay schedule. Could you fix that?